apps/src/javalab/JavalabEditor.jsx

```diff
@@ -468,17 +474,24 @@ class JavalabEditor extends React.Component {
             isRtl={false}
             onClick={this.props.handleVersionHistory}
           />
+          <PaneButton
+            id="javalab-editor-save"
+            iconClass="fa fa-check-circle"
+            onClick={onCommitCode}
+            headerHasFocus={true}
```

Comment:
what does this prop do?

also, style nit: for boolean props that are true, you can remove `={true}` (e.g., `<PaneButton headerHasFocus/>` will do the same thing)

Reply:
I think it affects the color of the button and I can update those boolean props!